cmd/download_grants_gov_db/handler_test.go

```diff
@@ -0,0 +1,222 @@
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"testing"
+	"time"
+
+	goenv "github.com/Netflix/go-env"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/credentials"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+	"github.com/go-kit/log"
+	"github.com/johannesboyne/gofakes3"
+	"github.com/johannesboyne/gofakes3/backend/s3mem"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func setupLambdaEnvForTesting(t *testing.T) {
+	t.Helper()
```

Comment:
`t.Helper()` is added to functions called during test that aren't actually under test themselves. It helps keep test output readable. From the [docs](https://pkg.go.dev/testing#T.Helper):
> Helper marks the calling function as a test helper function. When printing file and line information, that function will be skipped. Helper may be called simultaneously from multiple goroutines.